### PR TITLE
docs: Fixing main `README.md` broken link for `flutterfire_ui`

### LINKED ISSUE #10459: [📚] Flutterfire github repo's `readMe` has broken link
https://github.com/firebase/flutterfire#documentation

Screenshot 2023-02-15 at 5 56 10 PM

Tries to access: https://github.com/firebase/flutterfire/blob/master/packages/flutterfire_ui/README.md which shows 404 page not found.

### PR DESCRIPTION
**Pull request by cgalo**
## Description
Removed reference to `flutterfire_ui` directory, and added list of UI related packages  (per [comment](https://github.com/firebase/flutterfire/pull/10509#pullrequestreview-1315474381)).

## Related Issues
This pull request resolves #10459 

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by cgalo** [comment]
Hi @russellwheatley, I saw your comment on this stale [PR ](https://github.com/firebase/flutterfire/pull/10509). I tried addressing your [comment ](https://github.com/firebase/flutterfire/pull/10509#pullrequestreview-1315474381). If you could please review this change and let me know if there are any additional `ui` packages you would want me to reference.

File: README.md [hunk 1]
```diff
@@ -38,7 +38,11 @@ and open source.
 
 - [Add Firebase to your Flutter app](https://firebase.google.com/docs/flutter/setup)
 - [Available plugins](https://firebase.google.com/docs/flutter/setup#available-plugins)
-- [FlutterFire UI](./packages/flutterfire_ui/README.md) (beta)
+- FlutterFire UIs (beta)
```
Comment (lesnitsky) [inline README.md:41]:
```suggestion
- Firebase UI
```

File: README.md [hunk 2]
```diff
@@ -38,7 +38,11 @@ and open source.
...
+  - [Auth UI](./packages/firebase_ui_auth/README.md)
```
Comment (lesnitsky) [inline README.md:42]:
```suggestion
  - [Firebase UI for Authentication](./packages/firebase_ui_auth/README.md)
```

File: README.md [hunk 3]
```diff
@@ -38,7 +38,11 @@ and open source.
...
+  - [Database UI](./packages/firebase_ui_database/README.md)
```
Comment (lesnitsky) [inline README.md:43]:
```suggestion
  - [Firebase UI for Realtime Database](./packages/firebase_ui_database/README.md)
```

File: README.md [hunk 4]
```diff
@@ -38,7 +38,11 @@ and open source.
...
+  - [Firestore UI](./packages/firebase_ui_firestore/README.md)
```
Comment (lesnitsky) [inline README.md:44]:
```suggestion
  - [Firebase UI for Cloud Firestore](./packages/firebase_ui_firestore/README.md)
```

File: README.md [hunk 5]
```diff
@@ -38,7 +38,11 @@ and open source.
...
+  - [OAuth UI](./packages/firebase_ui_oauth/README.md)
```
Comment (lesnitsky) [inline README.md:45]:
```suggestion
```

Comment (lesnitsky) [reply README.md:45]: this one is rarely used directly and there are notes on `firebase_ui_auth` about `firebase_ui_oauth` installation

**Review by lesnitsky — CHANGES_REQUESTED** [review]

**Review by russellwheatley — APPROVED** [review]

**Pull request merged**